### PR TITLE
Added MySQL/YAML achievement support

### PR DESCRIPTION
**Pull request by Feeliiix**
The title says all

### CODE REVIEW

**Comment by Feeliiix** [comment]
Dont merg it found a big bugg

**Comment by Feeliiix** [comment]
buggs fixed. could be added

**Comment by Janhektor** [comment]
Hi Felix!

Thanks for your nice work.
There is only one thing to change: Please observe the code-style guidelines.
Example:
try{
Please insert a white-space to make it look like this:
try {

If you're using Eclipse, simply press STRG + SHIFT + F.

Thank you!

**Comment by Janmm14** [comment]
Probably this will not get pushed as its on the reserved list: https://github.com/Janhektor/Community/blob/master/RESERVED.md

**Comment by Feeliiix** [comment]
Ohh your right. Updated code.

File: src/de/janhektor/community/achievements/data/MySQLAchievementDataManager.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        this.asyncMySQL = Main.getInstance().getAsyncMySQL();
+    }
+
+    @Override
+    public List<String> getReachedAchievements(Player player) {
+        List<String> reachedAchievements = new ArrayList<>();
+        try {
+            PreparedStatement dataStatement = this.asyncMySQL.prepare("SELECT * FROM achievements WHERE UUID = ?");
+            dataStatement.setString(1, player.getUniqueId().toString());
+            ResultSet data = dataStatement.executeQuery();
```
Comment (Janhektor) [inline src/de/janhektor/community/achievements/data/MySQLAchievementDataManager.java]: Please work asynchronously.

**Comment by Feeliiix** [comment]
It was planned but it had buggs. Now its fixed! :D

**Comment by Feeliiix** [comment]
Will create a new pull with 1 commit... to many commits

**Pull request closed without merging**